Base/AutoUpdateChecker.cs

```diff
@@ -9,16 +9,25 @@ namespace MobiFlight.UpdateChecker
 {
     static class AutoUpdateChecker
     {
-        static string mobiFlightInstaller = "MobiFlight-Installer.exe";
-        static int UpdateCheckTimeoutInMs = 5000;
-        public static void CheckForUpdate(bool force = false, bool silent = false)
+        static readonly string mobiFlightInstaller = "MobiFlight-Installer.exe";
+        static readonly int UpdateCheckTimeoutInMs = 5000;
+        public static void CheckForUpdate(bool silent = false)
```

Comment:
you missed one call to CheckForUpdate in line 610. It should remove true now entirely.

Reply:
Oooooops.